packages/wallet-connectors/src/WalletConnect.ts

```diff
@@ -401,22 +573,28 @@ export class WalletConnectConnector implements WalletConnector {
      * if the dApp doesn't have its own {@link https://cloud.walletconnect.com WalletConnect Cloud} project.
      * @param delegate The object to receive events emitted by the client.
      * @param network The network/chain that connected accounts must live on.
+     * @param connectionScope The scope of the connections, i.e. which methods and events to request permission for in the wallet.
+     * @param [modalConfig] The configuration of the modal for connecting to the mobile wallet. Defaults to the default invocation of {@linkcode createWalletConnectModalConfig}
      */
     static async create(
         signClientInitOpts: SignClientTypes.Options,
         delegate: WalletConnectionDelegate,
-        network: Network
+        network: Network,
+        connectionScope: WalletConnectConnectionScope,
```

Comment:
Consider providing a default value (that includes the full "scope") to prevent this from being a breaking change.